Сверстаны два экрана и переходы между ними.

Bjjyx opened the pull request:
Two screens and transitions between them are laid out.

Bjjyx committed:
Сверстаны два экрана и переходы между ними.

Two screens and transitions between them are laid out.

Commit:
Коммит для сохранения.
Небольшие изменения в коде:
-Изменены id некоторых элементов
-Больше кода перенесено в style из разметки
-Добавлены шрифты

Save commit
Little changes in the code:
-The ids of some elements have been changed
-More code has been moved to style
-Added fonts

IliaAnK commented on app/src/main/res/values/style.xml:
```diff
@@ -0,0 +1,46 @@
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="titleTextColor">@color/white</item>
+        <item name="android:paddingVertical">12dp</item>
```
Нужно удалить

IliaAnK commented on app/src/main/res/values/style.xml:
```diff
@@ -0,0 +1,46 @@
+    </style>
+    <style name="toolbar" parent="Widget.AppCompat.Toolbar">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">wrap_content</item>
```
56dp или `?attr/actionBarSize`

IliaAnK commented on app/src/main/res/layout/activity_settings.xml:
```diff
@@ -0,0 +1,49 @@
+        app:navigationIcon="@drawable/arrow_back"
+        app:title="@string/settings"/>
+
+    <androidx.appcompat.widget.SwitchCompat
```
Для этой view нужно добавить отступ сверху, то есть `android:layout_marginTop="24dp"`

Bjjyx committed:
Коммит для сохранения.

Переделаны некоторые цвета
Добавлена "темная тема"
Добавлены темы
Исправлен размер toolbar
Исправлен отступ от toolbar в activity_settings
Исправлены все найденные размеры
Цвета используются из ресурсов.

Save commit.

Some colors have been redone
Added "dark theme"
Added themes
Fixed the size of the toolbar
Fixed the indentation from the toolbar in activity_settings
Fixed all found sizes
Colors are used from resources.

Bjjyx committed:
Коммит для сохранения.

Доработан switch в activity_settings (смена цвета согласно макету)
Кнопки "Поделиться", "Написать в поддержку", "Пользовательское соглашение" теперь работают, однако кнопка "Поделиться" требует доработки.

Save commit.

Completed switch in activity_settings (color change as in the layout)
The "Share", "Write to support", and "User Agreement" buttons are now working, but the "Share" button needs to be improved.

Bjjyx committed:
Коммит для сохранения.

Доработан switch в activity_settings (смена цвета согласно макету)
Кнопки "Поделиться", "Написать в поддержку", "Пользовательское соглашение" теперь работают, однако кнопка "Поделиться" требует доработки.

Save commit.

Completed switch in activity_settings (color change as in the layout)
The "Share", "Write to support", and "User Agreement" buttons are now working, but the "Share" button needs to be improved.

Bjjyx committed:
Коммит для сохранения.

Сверстан search_activity
Прописана первоначальная логика в search_activity

Save commit.

search_activity has been made
The initial logic is spelled out in search_activity

Bjjyx committed:
Коммит для сохранения.

Добавлен код для сохранения написанного пользователем текста в поле editText

Save commit.

search_activity has been made
Added code to save user's text in the editText
